hibernate-core/src/main/java/org/hibernate/dialect/Oracle8iDialect.java

```diff
@@ -420,7 +420,7 @@ else if ( initialValue > 0 && incrementSize < 0 ) {
 		else {
 			return
 				String.format(
-						"%s start with %d increment by %d",
+						"%s start with %d increment by  %d",
```

Comment:
is the above space char intended?

Reply:
@NathanQingyangXu idk. You removed it on your PR. This is a simple revert of all changes of you commit.
I can remove the space char again if you want.